homeassistant/components/shelly/sensor.py

```diff
@@ -127,7 +127,7 @@
 
 REST_SENSORS = {
     ("rssi"): RestAttributeDescription(
-        name="Rssi",
+        name="RSSI",
         unit="dB",
```

Comment:
wait for https://github.com/home-assistant/core/pull/40436 and change this to use the new const

Reply:
Can be the other way around as well.
The first merged will trigger an update for the other one.